build.gradle.kts

```diff
@@ -13,3 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+plugins {
+  `java-library`
+  `maven-publish`
+}
+
+repositories {
+  mavenCentral()
+}
+
+group = "org.projectnessie"
+
+version = file("version.txt").readText().trim()
+
+description = "catalog-migration-tool"
```

Comment:
nit: does this clash with `iceberg-catalog-migrator` project name?

Reply:
I thought it is a description and need not be same as project name.
I will make it same in a follow up PR after rebasing #2